app/views/shared/_order_detail_cell.html.haml

```diff
@@ -6,6 +6,10 @@
     - if show_reservation && od.reservation
       %br
       %em= od.reservation
+
+    - if od.product.offline?
```

Comment:
That's a good point. I could definitely see it being confusing. While it might be nice for the "New/In Process" tab, it's probably not worth the confusion of having it in both places and the complexity of only showing it in one.